### ISSUE TITLE
Is this crate still functional?

### ISSUE THREAD

**Issue by nekename** [adierkens/loupedeck-rs#1]
Hey there,

I'm the developer of [OpenDeck](https://github.com/ninjadev64/OpenDeck), and I'm looking to expand its device support to include Loupedeck.

I was wondering if this library is still functional, or if I should turn to binding to Node.js or Python code to use the Loupedeck libraries available for those languages?

Many thanks

**Comment by levyian** [comment]
@ninjadev64, would love to see this too. Have you made any headway? I'm willing to help with coding and testing.

**Comment by nekename** [comment]
No, I haven't. Feel free to join the Discord and I can give you an overview of implementing device support in a plugin.
